components/server/go/pkg/lib/cookie.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package lib
+
+import "regexp"
+
+func CookieNameFromDomain(domain string) string {
+	// replace all non-word characters with underscores
+	derived := regexp.MustCompile(`[\W_]+`).ReplaceAllString(domain, "_")
+	return "_" + derived + "_jwt2_"
```

Comment:
Once the file is detached, we may still forget to update the file

Reply:
How about using a tmpl file and render it in `.ts` and `.go` files

Reply:
> Once the file is detached, we may still forget to update the file

Could you explain what you mean with that? :confused: 
 
The reasoning behind this change is:
 - cookie name is generated in `installer`
 - consumed by `server`
 - consumed by `ws-proxy` tests

If we change the pattern here, and forget to update the check in ws-proxy, it will fail.

Is there a piece in the chain I'm missing? :thinking:

Reply:
I see